pallets/moonbeam-orbiters/src/weights.rs

```diff
@@ -143,61 +141,63 @@ impl WeightInfo for () {
 	// Storage: Balances Reserves (r:1 w:0)
 	#[rustfmt::skip]
 	fn collator_add_orbiter() -> Weight {
-		(31_392_000 as Weight)
-			.saturating_add(RocksDbWeight::get().reads(2 as Weight))
-			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
+		Weight::from_ref_time(46_047_000 as u64)
+			.saturating_add(RocksDbWeight::get().reads(2 as u64))
+			.saturating_add(RocksDbWeight::get().writes(1 as u64))
 	}
 	// Storage: MoonbeamOrbiters CollatorsPool (r:1 w:1)
 	#[rustfmt::skip]
 	fn collator_remove_orbiter() -> Weight {
-		(26_451_000 as Weight)
-			.saturating_add(RocksDbWeight::get().reads(1 as Weight))
-			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
+		Weight::from_ref_time(39_380_000 as u64)
+			.saturating_add(RocksDbWeight::get().reads(1 as u64))
+			.saturating_add(RocksDbWeight::get().writes(1 as u64))
 	}
 	// Storage: MoonbeamOrbiters CollatorsPool (r:1 w:1)
 	#[rustfmt::skip]
 	fn orbiter_leave_collator_pool() -> Weight {
-		(26_463_000 as Weight)
-			.saturating_add(RocksDbWeight::get().reads(1 as Weight))
-			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
+		Weight::from_ref_time(39_455_000 as u64)
+			.saturating_add(RocksDbWeight::get().reads(1 as u64))
+			.saturating_add(RocksDbWeight::get().writes(1 as u64))
 	}
 	// Storage: MoonbeamOrbiters MinOrbiterDeposit (r:1 w:0)
 	// Storage: Balances Reserves (r:1 w:1)
 	// Storage: System Account (r:1 w:1)
+	// Storage: MoonbeamOrbiters RegisteredOrbiter (r:0 w:1)
```

Comment:
Interesting that there is now a new db read

Reply:
This is weird. It added a write for `RegisteredOrbiter`, which I would expect a fn `register_orbiter` to write to. I wonder why it wasn't there previously...

Reply:
It's normal, @alan asked me to add a storage map in the orbiter pallet to indicate which accounts are orbiters to simplify the use for the outside world, but this storage is useless from the blockchain point of view.